index.html

```diff
@@ -1,534 +1,1010 @@
-            Avocatul Sebastian Nagy Cabinetul oferă servicii juridice în
-            diverse domenii de drept, adaptate nevoilor specifice
-            dumneavoastră.
+            Pasiunea pentru un act de justiție de calitate l-a împins pe
+            Sebastian Nagy sa fie prezent în mod constant în presa locală și
+            națională, contribuind cu expertiza sa la educația juridică a
+            intregii societati.
           </p>
         </div>
       </div>
-      <img src="img/law-3--lazy.jpg" data-src="img/law-2.jpg" alt="Plant" class="features__img lazy-img" />
-      <div class="features__feature features__feature-m">
-        <div class="features__icon features__icon-m">
-          <svg>
-            <use xlink:href="img/icons.svg#icon-trending-up"></use>
-          </svg>
-        </div>
-        <div class="features__deatail-cont">
+    </section>
+
+    <section class="section" id="servicii">
+      <div class="section__title">
+        <h2 class="section__description">Soluții există</h2>
+        <h3 class="section__header">
+          Servicii juridice de încredere în Cluj-Napoca
+        </h3>
+      </div>
+
+      <div class="features">
+        <img
+          src="img/law-1.jpg"
+          data-src="img/law-1.jpg"
+          alt="Computer"
+          class="features__img"
+        />
+        <div class="features__feature">
+          <div class="features__icon">
+            <svg>
+              <use xlink:href="img/icons.svg#icon-monitor"></use>
+            </svg>
+          </div>
           <h5 class="features__header">Drept Penal</h5>
           <p>
             Apărare și reprezentare în dosare penale, asigurându-se că
-            drepturile și interesele clienților săi sunt protejate cu
-            fermitate
+            drepturile și interesele clienților săi sunt protejate cu fermitate.
           </p>
         </div>
-      </div>
-      <img src="img/law-3--lazy.jpg" data-src="img/law-3.jpg" alt="Credit card" class="features__img lazy-img" />
 
-      <div class="features__feature features__feature-m">
-        <div class="features__icon features__icon-m">
-          <svg>
-            <use xlink:href="img/icons.svg#icon-credit-card"></use>
-          </svg>
+        <div class="features__feature">
+          <div class="features__icon">
+            <svg>
+              <use xlink:href="img/icons.svg#icon-trending-up"></use>
+            </svg>
+          </div>
+          <h5 class="features__header">Dreptul Familiei</h5>
+          <p>
+            Soluționarea problemelor legate de divorț, custodie, adopție și alte
+            aspecte legate de dreptul familiei.
+          </p>
         </div>
-        <div class="features__deatail-cont">
+        <img
+          src="img/law-2.jpg"
+          data-src="img/law-2.jpg"
+          alt="Plant"
+          class="features__img"
+        />
+
+        <img
+          src="img/law-3.jpg"
+          data-src="img/law-3.jpg"
+          alt="Credit card"
```

Comment:
Please add meaningful **alt text** to all images. This is very important for accessibility reasons and SEO reasons. Here you can use "Avocat definitiv Cluj-Napoca Sebastian Nagy servicii juridice".